src/terminal/parser/telemetry.cpp

```diff
@@ -221,7 +221,9 @@ void TermTelemetry::WriteFinalTraceLog() const
                                       TraceLoggingUInt32(_uiTimesUsed[SGR], "SGR"),
                                       TraceLoggingUInt32(_uiTimesUsed[DECSC], "DECSC"),
                                       TraceLoggingUInt32(_uiTimesUsed[DECRC], "DECRC"),
+                                      TraceLoggingUInt32(_uiTimesUsed[DECSET], "SM"),
                                       TraceLoggingUInt32(_uiTimesUsed[DECSET], "DECSET"),
+                                      TraceLoggingUInt32(_uiTimesUsed[DECRST], "RM"),

```

Comment:
```suggestion
                                      TraceLoggingUInt32(_uiTimesUsed[RM], "RM"),
```

(if I understand this correctly)

Reply:
I don't know how I missed that. Thanks!